code/modules/power/supermatter/supermatter_crate.dm

```diff
@@ -1,7 +0,0 @@
-/datum/supply_packs/engineering/engine/supermatter_shard

```

Comment:
I originally put this here because supermatter code was 100% self-contained in this folder. People have added it to other random parts of the code already though so i guess it isn't that big of a deal